tests/console/validate_vnc_avail.pm

```diff
@@ -0,0 +1,64 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Validate that after remote installation, the VNC connection
+# between client and host, is available without the use of vnc password.
+#
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use base "opensusebasetest";
+use strict;
+use warnings;
+use testapi;
+use lockapi;
+use mmapi;
+use y2lan_restart_common "close_xterm";
+
+# Returns the first leased IP, in /var/lib/dhcp/db/dhcpd.leases file
+sub get_lease_ipaddr {
+    my $lease_ip;
+    my $dhcp_leases_file = script_output("cat /var/lib/dhcp/db/dhcpd.leases\n");
+    for my $line (split /\n/, $dhcp_leases_file) {
+        if ($line =~ /^lease\s+\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}/) {
+            (undef, $lease_ip) = split(/\s/, $line);
```

Comment:
In the line above you can use named capture regex for the ip and add  'lease\s+' and '\s+}' so you can avoid that magic code with the undef.

Reply:
I copied the particular code from another module, but your proposal makes it better understandable. Done.